resize.php

```diff
@@ -0,0 +1,71 @@
+<?php
+require(dirname(__FILE__).'/config/config.inc.php');
+$url = filter_input(INPUT_GET, 'url');
```

Comment:
```suggestion
$url = filter_input(INPUT_GET, 'url');
$matches = [];
```

I think it's a bit easier to read if `$matches` is declared before although php does not force us to do it.